StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsSessionManifest.swift

```diff
@@ -45,6 +45,12 @@ struct FinancialConnectionsSessionManifest: Decodable {
         case unparsable
     }
 
+    enum ManualEntryMode: String, SafeEnumCodable, Equatable {
+        case automatic
+        case custom
+        case unparsable
+    }
+
```

Comment:
backend model to double check:
- https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/api/resource/enums/manual_entry_mode_api_enum.rb#L5-15